config/manifests/bases/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -33,6 +33,11 @@ spec:
       - description: Group of Nodes to enable RTE on
         displayName: Group of nodes to enable RTE on
         path: nodeGroups
+      - description: Set the topology info refresh period. Use explicit 0 to disable.
+        displayName: Topology info refresh period setting
+        path: nodeGroups[0].config.infoRefreshPeriod
```

Comment:
Similar to my query above, would really appreciate some clarification here.

Reply:
agreed